### PR TITLE
Organize imports

### LINKED ISSUE #2107: Organize imports after linting the code.
After #2045 is finished,  organize `eslintrc`, sort `eslintignore`, order the imports and comments,  and remove unnecessary imports

### PR DESCRIPTION
**Pull request by mauriciovieira**
1. remove unused imports
2. organize imports (sort and unify comments)
3. remove unused eslint-disable

closes #2107

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@apinf/developers please review

**Comment by mauriciovieira** [comment]
Notice I standardized the first comments with:

```
// Meteor packages imports

// Npm packages imports

// Collection imports

// APINF imports
```

The imports follow this order unless breaks eslint

**Comment by brylie** [comment]
I am wondering whether removing all of these explicit imports is the 'right thing to do'. In other words, it may be useful, and at some point necessary, to explicitly import Meteor components. Consistency with imports also helps our developers to better understand how things are connected, and is a common practice with other frameworks that don't have such a 'magical' environment as Meteor.

**Comment by brylie** [comment]
The Meteor Guide also recommends using explicit imports for Meteor libraries.

**Comment by mauriciovieira** [comment]
What I mind mostly is the consistency. Before we started to run eslint (a month ago) we had meteor imports only in some files.

**Comment by brylie** [comment]
Yes, for the same reason we had lint in only a few files. In effect, the strict import has not been a problem for the duration of our development, and has only recently been caught by the linter. Hence, the work in progress towards consistent use of explicit imports.

**Comment by mauriciovieira** [comment]
No problem, I'll write some script to put them back (removing using sed was easy). 
I spent a few hours yesterday sorting and commenting all files and won't like to waste this work.

**Comment by brylie** [comment]
Cool. I am not trying to give you grief. I'm just clarifying a direction we are heading, based on the recommendation of the Meteor Guide.

**Comment by mauriciovieira** [comment]
Use node to reinsert the import statements http://nicolashery.com/parse-data-files-using-nodejs-streams/ (https://github.com/nicolashery/example-stream-parser/blob/master/lib/parser.js)

**Comment by brylie** [comment]
Is this ready for review? It might be easier to just create a new branch, since most of the changes were unnecessary. Also, I am not sure we should add 'meteor' to the eslint environment, since it might discourage developers from being explicit about Meteor imports.

**Comment by mauriciovieira** [comment]
I prefer not undoing the manual work I've done to comment and sort all imports.

I wanted to work on a script to automatically fix the meteor no-undef. This script might be useful. I am still working on that.

**Comment by mauriciovieira** [comment]
@brylie I rebased the commits. 

I standardized the first comments with:

```
// Meteor packages imports

// Meteor contributed packages imports

// Npm packages imports

// Collection imports

// APINF imports
```

Also, I noticed some broken references in the develop branch and fixed them. 

Please review.

**Comment by mauriciovieira** [comment]
'Add meteor env to eslintrc' is not a suitable description for this work. It's better to call it 'Organize imports'. I renamed it

File: api_docs/client/swaggerUI/swaggerUiContent/swaggerUiContent.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
 import { Template } from 'meteor/templating';
 
-import Apis from '/apis/collection';
+// Meteor contributed packages imports
 import ApiKeys from '/api_keys/collection';
```
Comment (marla-singer) [inline api_docs/client/swaggerUI/swaggerUiContent/swaggerUiContent.js]: Hm. Looks like it's `// Collection imports` not `// Meteor contributed packages imports`

File: api_docs/files/collection/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Meteor packages imports
 import { FileCollection } from 'meteor/vsivsi:file-collection';
```
Comment (marla-singer) [inline api_docs/files/collection/index.js]: `Meteor contributed packages imports` here

File: api_keys/collection/schema.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { SimpleSchema } from 'meteor/aldeed:simple-schema';
 
+// Npm packages imports
 import ApiKeys from './';
```
Comment (marla-singer) [inline api_keys/collection/schema.js]: `Collection imports` here

File: apis/client/profile/intro/intro.js [hunk 4]
```diff
@@ -1,11 +1,10 @@
-// Allow use of 'document' as global object
-/* global document */
+// Meteor packages imports
+import { Template } from 'meteor/templating';
 
 // Meteor imports
 import { TAPi18n } from 'meteor/tap:i18n';
```
Comment (marla-singer) [inline apis/client/profile/intro/intro.js]: `Meteor contributed packages imports` here

File: apis/client/profile/settings/details/autoform.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
-// Meteor package import
+// Meteor packages imports
 import { AutoForm } from 'meteor/aldeed:autoform';
```
Comment (marla-singer) [inline apis/client/profile/settings/details/autoform.js]: `Meteor contributed packages imports` here

File: apis/client/profile/status/convert_status_code.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-// Meteor packages import
+// Meteor packages imports
 import { TAPi18n } from 'meteor/tap:i18n';
```
Comment (marla-singer) [inline apis/client/profile/status/convert_status_code.js]: `Meteor contributed packages imports` here

File: apis/collection/schema.js [hunk 7]
```diff
@@ -1,4 +1,7 @@
+// Meteor packages imports
 import { SimpleSchema } from 'meteor/aldeed:simple-schema';
```
Comment (marla-singer) [inline apis/collection/schema.js]: `Meteor contributed packages imports` here

File: apis/collection/server/permissions.js [hunk 8]
```diff
@@ -1,7 +1,10 @@
-import _ from 'lodash';
-
+// Meteor packages imports
 import { Roles } from 'meteor/alanning:roles';
```
Comment (marla-singer) [inline apis/collection/server/permissions.js]: `Meteor contributed packages imports` here

File: apis/flags/collection/permissions.js [hunk 9]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Roles } from 'meteor/alanning:roles';
```
Comment (marla-singer) [inline apis/flags/collection/permissions.js]: `Meteor contributed packages imports` here

File: apis/logo/collection/index.js [hunk 10]
```diff
@@ -1,3 +1,4 @@
+// Meteor packages imports
 import { FileCollection } from 'meteor/vsivsi:file-collection';
```
Comment (marla-singer) [inline apis/logo/collection/index.js]: `Meteor contributed packages imports` here

File: apis/server/methods/status.js [hunk 11]
```diff
@@ -1,5 +1,7 @@
-import { check } from 'meteor/check';
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+
 
 import Apis from '/apis/collection';
```
Comment (marla-singer) [inline apis/server/methods/status.js]: `Collection imports` here

File: users/collection/users_permissions.js [hunk 12]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
 
+
```
Comment (marla-singer) [inline users/collection/users_permissions.js]: New line

File: backlog/collection/index.js [hunk 13]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Mongo } from 'meteor/mongo';
 
+
```
Comment (marla-singer) [inline backlog/collection/index.js]: Excessive new line

File: core/client/error_templates/go_back_button/goBackButton.js [hunk 14]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Template } from 'meteor/templating';
 
+
```
Comment (marla-singer) [inline core/client/error_templates/go_back_button/goBackButton.js]: Excessive new line

File: dashboard/client/charts/table/table.js [hunk 15]
```diff
@@ -1,5 +1,7 @@
-import { Template } from 'meteor/templating';
+// Meteor packages imports
 import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+
```
Comment (marla-singer) [inline dashboard/client/charts/table/table.js]: Excessive new line

File: feedback/client/list/list.js [hunk 16]
```diff
@@ -1,7 +1,9 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
-import { Modal } from 'meteor/peppelg:bootstrap-3-modal';
 import { Template } from 'meteor/templating';
 
+import { Modal } from 'meteor/peppelg:bootstrap-3-modal';
+
 import Feedback from '../../collection';
```
Comment (marla-singer) [inline feedback/client/list/list.js]: `Collection imports` here

File: feedback/collection/schema.js [hunk 17]
```diff
@@ -1,4 +1,6 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
+
 import { SimpleSchema } from 'meteor/aldeed:simple-schema';
 import Feedback from './';
```
Comment (marla-singer) [inline feedback/collection/schema.js]: `Collection imports` here

File: home/client/footer/homeFooter.js [hunk 18]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Template } from 'meteor/templating';
 
+
```
Comment (marla-singer) [inline home/client/footer/homeFooter.js]: Excessive new line

File: proxy_backends/client/modal_form/remove_selected_proxy/remove_selected_proxy.js [hunk 19]
```diff
@@ -0,0 +1,28 @@
+// Meteor packages imports
+import { Template } from 'meteor/templating';
```
Comment (marla-singer) [inline proxy_backends/client/modal_form/remove_selected_proxy/remove_selected_proxy.js]: @mauriciovieira `removed_selected_proxy` folder was deleted

File: status/client/status.js [hunk 20]
```diff
@@ -1,6 +1,8 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
 import { Template } from 'meteor/templating';
 
+
```
Comment (marla-singer) [inline status/client/status.js]: Excessive new line

File: status/server/status.js [hunk 21]
```diff
@@ -1,5 +1,7 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
 
+
```
Comment (marla-singer) [inline status/server/status.js]: Excessive new line

File: users/client/profile/profile.js [hunk 22]
```diff
@@ -1,6 +1,8 @@
+// Meteor packages imports
 import { Meteor } from 'meteor/meteor';
 import { Template } from 'meteor/templating';
 
+
```
Comment (marla-singer) [inline users/client/profile/profile.js]: Excessive new line

File: api_docs/client/swaggerUI/swaggerUiContent/swaggerUiContent.js (hunk 1 above)
Comment (mauriciovieira) [inline api_docs/client/swaggerUI/swaggerUiContent/swaggerUiContent.js]: 🎉

File: api_keys/collection/schema.js (hunk 3 above)
Comment (mauriciovieira) [inline api_keys/collection/schema.js]: 👍

File: apis/client/profile/intro/intro.js (hunk 4 above)
Comment (mauriciovieira) [inline apis/client/profile/intro/intro.js]: 👍

File: apis/server/methods/status.js (hunk 11 above)
Comment (mauriciovieira) [inline apis/server/methods/status.js]: 🎉

File: users/collection/users_permissions.js (hunk 12 above)
Comment (mauriciovieira) [inline users/collection/users_permissions.js]: 👍

File: core/client/error_templates/go_back_button/goBackButton.js (hunk 14 above)
Comment (mauriciovieira) [inline core/client/error_templates/go_back_button/goBackButton.js]: 👍

File: dashboard/client/charts/table/table.js (hunk 15 above)
Comment (mauriciovieira) [inline dashboard/client/charts/table/table.js]: 👍

File: home/client/footer/homeFooter.js (hunk 18 above)
Comment (brylie) [inline home/client/footer/homeFooter.js]: :boom:

File: status/client/status.js (hunk 20 above)
Comment (brylie) [inline status/client/status.js]: :hammer:

File: status/server/status.js (hunk 21 above)
Comment (brylie) [inline status/server/status.js]: :last_quarter_moon_with_face:

File: users/client/profile/profile.js (hunk 22 above)
Comment (brylie) [inline users/client/profile/profile.js]: :pager:

File: feedback/client/list/list.js (hunk 16 above)
Comment (mauriciovieira) [inline feedback/client/list/list.js]: 👍

File: feedback/collection/schema.js (hunk 17 above)
Comment (mauriciovieira) [inline feedback/collection/schema.js]: 🎉

File: home/client/footer/homeFooter.js (hunk 18 above)
Comment (mauriciovieira) [inline home/client/footer/homeFooter.js]: ✌️

File: proxy_backends/client/modal_form/remove_selected_proxy/remove_selected_proxy.js (hunk 19 above)
Comment (mauriciovieira) [inline proxy_backends/client/modal_form/remove_selected_proxy/remove_selected_proxy.js]: Oh, thanks for catching this :-)

File: status/client/status.js (hunk 20 above)
Comment (mauriciovieira) [inline status/client/status.js]: :1st_place_medal:

File: status/server/status.js (hunk 21 above)
Comment (mauriciovieira) [inline status/server/status.js]: 👀

File: users/client/profile/profile.js (hunk 22 above)
Comment (mauriciovieira) [inline users/client/profile/profile.js]: 👀

**Comment by mauriciovieira** [comment]
@marla-singer Thank you so much for catching these errors. 
Please review again.

**Pull request merged**